.env.example

```diff
@@ -6,7 +6,7 @@
 # Instead, copy it, remove the .example extension and provide your custom values
 # For git password please use your generated git token, not a clear text password
 
-VSIX_FILE_NAME='konveyor-v0.0.10.vsix'
+VSIX_FILE_NAME='konveyor-v0.0.11.vsix'
 DEFAULT_VSIX_DOWNLOAD_URL= 'https://github.com/konveyor/editor-extensions/releases/download/v0.0.10/konveyor-v0.0.10.vsix'
```

Comment:
DEFAULT_VSIX_DOWNLOAD_URL also needs to be updated to 0.0.11

Reply:
Solved in 9b3a80ff73f65465249a525df893579b71351234